docker-compose.yml

```diff
@@ -0,0 +1,16 @@
+version: '3.1'
+
+services:
+  db:
+    image: mysql:latest
```

Comment:
The **latest** tag is usually not reproducible or stable for your infrastructure requirements.

Today let's say the latest tag is **8.3.1**.
If you want to run this application in the future next year, then **latest** could be **9.0.0** or **10.0.0**.

This is why its usually a best practice to pin the docker image versions to a specific version, not latest.